pg_search/migrations/0001_enable.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.db import migrations
+
```

Comment:
I guess we probably will need to keep built-in `tsearch` as an optional feature, even if we're able to make this to default, because third-party database providers may not offer a database with `pg_search` linked in.